nifi-nar-bundles/nifi-extension-utils/nifi-record-utils/nifi-json-record-utils/src/main/java/org/apache/nifi/json/TokenParserFactory.java

```diff
@@ -18,13 +18,19 @@
 
 import com.fasterxml.jackson.core.JsonParser;
 import com.fasterxml.jackson.core.StreamReadConstraints;
-import com.fasterxml.jackson.databind.ObjectMapper;
 
 import java.io.IOException;
 import java.io.InputStream;
 
 public interface TokenParserFactory {
-    JsonParser getJsonParser(InputStream in) throws IOException;
-
-    ObjectMapper createCodec(boolean allowComments, StreamReadConstraints streamReadConstraints);
+    /**
+     * Get JSON Parser implementation for provided Input Stream with configured settings
+     *
+     * @param in Input Stream to be parsed
+     * @param streamReadConstraints Stream Read Constraints applied
+     * @param allowComments Whether to allow comments when parsing
+     * @return JSON Parser
+     * @throws IOException Thrown on failures ot read the Input Stream
```

Comment:
Instead of
`@throws IOException Thrown on failures ot read the Input Stream`
this should be
`@throws IOException Thrown on failures to read the Input Stream`